Services/ADT/classes/Types/Group/class.ilADTGroupActiveRecordBridge.php

```diff
@@ -10,8 +10,24 @@ protected function isValidADT(ilADT $a_adt)
 	{
 		return ($a_adt instanceof ilADTGroup);
 	}
-	
-	
+
+
+	/**
+	 * @inheritDoc
+	 */
+	public function getFieldValue($a_field_name) {

```

Comment:
My InterfaceCollector just dies when accessing this class since there are method which are not implemented. I assume this class isn't even user in ILIAS.